lib/experimental/modules/class-gutenberg-modules.php

```diff
@@ -154,65 +180,91 @@ private static function get_version_query_string( $version ) {
 			},
 			array()
 		);
 	}
 }
 
 /**
- * Registers a JavaScript module. It will be added to the import map.
+ * Registers the module if no module with that module identifier has already
+ * been registered.
  *
- * @param string           $module_identifier The identifier of the module. Should be unique. It will be used in the final import map.
- * @param string           $src               Full URL of the module, or path of the script relative to the WordPress root directory.
- * @param array            $dependencies      Optional. An array of module identifiers of the static and dynamic dependencies of this module. It can be an indexed array, in which case all the dependencies are static, or it can be an associative array, in which case it has to contain the keys `static` and `dynamic`.
- * @param string|bool|null $version           Optional. String specifying module version number. It is added to the URL as a query string for cache busting purposes. If SCRIPT_DEBUG is true, a timestamp is used. If it is set to false, a version number is automatically added equal to current installed WordPress version. If set to null, no version is added.
+ * @param string            $module_identifier The identifier of the module. Should be unique. It will be used in the final import map.
+ * @param string            $src               Full URL of the module, or path of the script relative to the WordPress root directory.
+ * @param array             $dependencies      Optional. An array of module identifiers of the dependencies of this module. The dependencies can be strings or arrays. If they are arrays, they need an `id` key with the module identifier, and can contain a `type` key with either `static` or `dynamic`. By default, dependencies that don't contain a type are considered static.
+ * @param string|false|null $version           Optional. String specifying module version number. Defaults to false. It is added to the URL as a query string for cache busting purposes. If SCRIPT_DEBUG is true, the version is the current timestamp. If $version is set to false, the version number is the currently installed WordPress version. If $version is set to null, no version is added.
  */
 function gutenberg_register_module( $module_identifier, $src, $dependencies = array(), $version = false ) {
 	Gutenberg_Modules::register( $module_identifier, $src, $dependencies, $version );
 }
 
 /**
- * Enqueues a JavaScript module. It will be added to both the import map and a
- * script tag with the "module" type.
+ * Marks the module to be enqueued in the page.
  *
- * @param string $module_identifier The identifier of the module. Should be unique. It will be used in the final import map.
+ * @param string $module_identifier The identifier of the module.
  */
 function gutenberg_enqueue_module( $module_identifier ) {
 	Gutenberg_Modules::enqueue( $module_identifier );
 }
 
+/**
+ * Unmarks the module so it is not longer enqueued in the page.
```

Comment:
small typo: `is not longer` -> `is no longer`

Reply:
Done in https://github.com/WordPress/gutenberg/pull/57231/commits/0a4d3557e9fb1da8366fa77b16fe9afabd2fadc5.